types/ApiRequests.ts

```diff
@@ -49,10 +54,22 @@ export interface UserPatches {
   passwordChange?: PasswordUpdate;
 }
 
+export interface FeedbackPatches extends Partial<Feedback> {
+  acknowledged?: boolean;
+}
+
 export interface PatchUserRequest {
   user: UserPatches;
 }
 
+export interface AddFeedbackRequest {
+  feedback: Feedback;
+}
+
+export interface PatchFeedbackRequest {
```

Comment:
If someone acknowledges feedback, unacknowledges it, then acknowledges it again, does the user get more points than they should? To avoid this situation, should we disallow un-acknowledging feedback?